packages/terra-clinical-header/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "peerDependencies": {
-    "react": "^16.4.2",
+    "react": "^16.8.5",
     "react-dom": "^16.4.2"
```

Comment:
This one (and item collection, item display, item view, and label value view) too

Reply:
https://github.com/cerner/terra-clinical/pull/476/commits/ca2253ebee49b5a308b5906f148ffb7a1af57ee7